website/_kb/user-guide/getting-started.md

```diff
@@ -53,7 +53,18 @@ You can combine visualizations and text into thematic & powerful dashboards. Add
 
 ![](/assets/images/docs/gifs/dashboards/dashboards.gif)
 
-## 5. Invite Colleagues
+## 5. Add an avatar
```

Comment:
Are we sure this is important enough to be part of the Getting Started guide?

Reply:
Redash isn't a social site (yet?). :) Let's keep it in the separate page for now.